Recipes/Recipes.API/Controllers/RecipesController.cs

```diff
@@ -38,4 +38,69 @@ public async Task<ActionResult<IEnumerable<Recipe>>> GetAllRecipes()
 
         return Ok(responseRecipes);
     }
+
+    [HttpGet("{recipeId:int}")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public async Task<ActionResult<Recipe>> GetRecipe(int recipeId)
+    {
+        RecipeEntity? recipe;
+        try
+        {
+            recipe = await recipesRepository.GetByIdAsync(recipeId);
+        }
+        catch (Exception)
+        {
+            return StatusCode(StatusCodes.Status500InternalServerError);
+        }
+
+        if (recipe is null) return NotFound();
+
+        var responseRecipe = new Recipe
+        {
+            Id = recipe.Id,
+            Title = recipe.Title,
+            AuthorId = recipe.AuthorId,
+            Description = recipe.Description,
+            CookingTime = recipe.CookingTime,
+            Servings = recipe.Servings,
+            UpdatedAt = recipe.UpdatedAt,
+            Ingredients = recipe.Ingredients,
+            Instructions = recipe.Instructions
+        };
+
+        return Ok(responseRecipe);
+    }
+
+    [HttpDelete("{recipeId:int}")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/DELETE#responses

We don't supply any additional information, so we could use a 204 NoContent response instead.

Reply:
All right, will do